node/pkg/adminrpc/adminserver.go

```diff
@@ -577,6 +578,137 @@ func wormholeRelayerSetDefaultDeliveryProvider(req *nodev1.WormholeRelayerSetDef
 	return v, nil
 }
 
+func evmCallToVaa(evmCall *nodev1.EvmCall, timestamp time.Time, guardianSetIndex, nonce uint32, sequence uint64) (*vaa.VAA, error) {
+	// First we turn the signature into an ABI object. Naturally, we do this by
+	// parsing the selector, then turning it into JSON, then parsing that back
+	// into an ABI object.
+	// Just remember the web3 mantra: the more string allocation, the better.
+	selector, err := abi.ParseSelector(evmCall.Signature)
+	if err != nil {
+		return nil, err
```

Comment:
Rather than just returning the error, could you do this (for all of these errors)?
`return nil, fmt.Errorf("failed to parse selector: %w", err)`